charts/rhtap-backing-services/templates/_copy-scripts.tpl

```diff
@@ -0,0 +1,25 @@
+{{/*
+
+  POD container spec to copy scripts.
+
+*/}}
+{{- define "backingServices.copyScripts" -}}
+- name: copy-scripts
```

Comment:
Not related to this PR specifically, but I'm wondering if we should use init-containers in that pattern instead.

Reply:
That makes sense, +1